lib/packages/core/src/__tests__/burstService.spec.ts

```diff
@@ -1,11 +1,50 @@
 import {BurstService} from '../service/burstService';
-import {HttpMockBuilder, HttpError, Http} from '@burstjs/http';
+import {Http, HttpError, HttpImpl, HttpMockBuilder, HttpResponse} from '@burstjs/http';
 import {BurstServiceSettings} from '../service/burstServiceSettings';
 import {createBurstService} from './helpers/createBurstService';
 
+class TestHttpClient implements Http {
+    delete(url: string): Promise<HttpResponse> {
+        return undefined;
+    }
+
+    get(url: string): Promise<HttpResponse> {
+        return undefined;
+    }
+
+    post(url: string, payload: any): Promise<HttpResponse> {
+        return undefined;
+    }
+
+    put(url: string, payload: any): Promise<HttpResponse> {
+        return undefined;
+    }
+
+}
 
 describe('BurstService', () => {
+    describe('constructor', () => {
```

Comment:
having more tests is always nice.... kept these ones, after I reverted the prior BrsVersion stuff